sorc/ncep_post.fd/MDL2P.f

```diff
@@ -796,7 +796,8 @@ SUBROUTINE MDL2P(iostatusD3D)
 !
             IF(gridtype == 'E')THEN
               DO J=JSTA,JEND
-                DO I=2,IM-MOD(J,2)
+!                DO I=2,IM-MOD(J,2)
+                DO I=ISTA,IEND
```

Comment:
suggest changing to 
J=JSTA_M,JEND_M
I=ISTA_M, IEND_M

This is because line 852 has PMID with dimension "I+1" and line 858 with dimension i-1